src/test_report.py

```diff
@@ -29,6 +29,7 @@ class TestReport:
 
     def __init__(self, configs, args):
         self._db_config = configs['db_configs'][args.db_config]
+        self._email_config = configs['email_configs'][args.email_config]
```

Comment:
@mgalka I basically started testing this to move variables to `pipeline.yaml`, in yaml will be easy to have lists and multiple config options for email reports as we have for build, etc. 

If I run the code exactly like I pushed here, I got this log (note: I know that it will not work at all, but I need to ensure that variables will be loaded, afterward I just need to update the code and not "import" variables from kernelci.conf anymore):

`kernelci-pipeline-test_report | Traceback (most recent call last):
kernelci-pipeline-test_report |   File "/home/kernelci/pipeline/test_report.py", line 124, in <module>
kernelci-pipeline-test_report |     status = opts.command(configs, opts)
kernelci-pipeline-test_report |   File "/home/kernelci/pipeline/test_report.py", line 117, in __call__
kernelci-pipeline-test_report |     generate_test_report = TestReport(configs, args)
kernelci-pipeline-test_report |   File "/home/kernelci/pipeline/test_report.py", line 32, in __init__
kernelci-pipeline-test_report |     self._email_config = configs['email_configs'][args.email_config]
kernelci-pipeline-test_report | KeyError: 'email_configs'
`
then if I check the container by itself, I can see the variables in `pipeline.yaml` in the container.

`docker exec -it kernelci-pipeline-test_report sh
cat config/pipeline.yaml
email_configs:
  staging:
    email_send_to: ['subscribers@from.email.reports', 'alex.pereira@gmail.com']
    email_send_from: 'kernelci-results-staging@groups.io'
    email_subject: 'Kernel CI Test Reports'
`

Do you have any idea about what I did wrong? 
Cheers o/

Reply:
@alexandrasp Maybe we could drop the YAML support for the time being?

Reply:
done